### ISSUE TITLE
Add ability to set API key without being administrator of the instance

### ISSUE THREAD

**Issue by winux1** [searx/searx#2532]
**Is your feature request related to a problem? Please describe.**
Not realy. See discussion #2527
**Describe the solution you'd like**
Please add ability to set api key on the settings page of searx  in the browser for every user

**Comment by return42** [comment]
Hm .. I am not familiar with flicker, but in general: is it really a good idea to upload private keys to a unknown server instance?

**Comment by winux1** [comment]
I agree with you @return42. This option have to be optional for every engine that needs an API key
